### DISCUSSION TITLE
settings file

### DISCUSSION THREAD

**Discussion by redsparrowgit** [TheLastBen/fast-stable-diffusion#816] [General]
I have a questions
1) which file is responsible for all the settings in gradio? I would like to copy it to not have to set everything every time in the future. 
NEW IDEA ( download/upload settings from gradio )

2) is it possible to ADD somewhere in Colab so that it is automatically added to the address ?__theme=dark ?

**Comment by TheLastBen** [comment]
the settings are in **"config.json"** in the "stable-diffusion-webui" folder.
as for the theme, if you set your windows them to dark, the webui will automatically switch to dark.

**Reply by redsparrowgit** [reply]
Thanks, i don`t use windows :)
